tests/hardware_tests/test_nav.py

```diff
@@ -0,0 +1,123 @@
+import time
```

Comment:
what is the function for?

Reply:
time is used to sleep the thread for instance, hand rotated by 20 degrees then sleep for 1 sec to settle the motion blur, otherwise detection model doesn't give good detections